ERCS/erc-7786.md

```diff
@@ -103,6 +103,12 @@ This event signals that a would-be sender has requested a message to be sent.
 
 If `outboxId` is present, post-processing MAY be required to send the message through the cross-chain channel.
 
+#### Mandatory and optional attributes
+
+The `attributes` parameter MUST contain as its first element the value `read(true)` or `read(false)`. `read(true)` means the message can be read by the application on the destination gateway. `read(false)` means the message by pushed by the destination gateway on the application via `executeMessage`(see below).
```

Comment:
1. if mandatory, why not just explicitly enforce at the function signature level? i.e. `sendMessage(dest, recv, payload, bool pullable, attributes)` 
2. personally, i think `read(true)` is not intelligible. I prefer one of `pullable`, `supports_pull`, `pull_based`, `push_based`. 

wdyt? @frangio @elliedavidson

Reply:
`pushBased` used in 15c1f1b7dea821026f012451b6191e1c93093c5a.